.readthedocs.yml

```diff
@@ -1,5 +1,17 @@
+# https://docs.readthedocs.io/en/stable/config-file/v2.html
+version: 2
+
+sphinx:
+  fail_on_warning: false
```

Comment:
This is ok. If you set it to True, RTD will report a failure if a warning is present, but will not generate the HTML.

I prefer to have the HTML on RTD even on warnings... so that you can check the result.
**And** have a separate GiHub action that will fail on errors.

I see there is an existing 'docs' jobs it's green even with one warning

```
 /home/runner/work/treq/treq/docs/api.rst:80: WARNING: more than one target found for cross-reference 'request': treq.request, treq.client.HTTPClient.request, treq.testing.treq.testing.StubTreq.request, treq.testing.RequestTraversalAgent.request
```

Reply:
I agree, if only I could figure out how to fix that warning.